### ISSUE TITLE
spawn ENOENT

### ISSUE THREAD

**Issue by legarspol** [qrpike/spotify-playlist-downloader#7]
Hi !
Your app works great, But at the end of the dl I get a: "Error: spawn ENOENT"
No idea why...

```
 - Downloading: [song's name] 
 -  - DONE:  [song's name] 
Error: spawn ENOENT
```

**Comment by killswitch** [comment]
I had the same issue https://github.com/qrpike/spotify-playlist-downloader/issues/2

**Comment by roshanr10** [comment]
I had the same issue and discovered that it is due to the metadata being written to the file. If you install FFMPEG, it should work much better. @legarspol @killswitch

**Comment by gtothesquare** [comment]
@roshanr10 Are you still being able to authenticate ?

**Comment by roshanr10** [comment]
@gtothesquare Yes, I am still able to authenticate and download playlists, although only on node v.10. I am currently seeing if it is possible to modify it to work on the latest version.

**Comment by gtothesquare** [comment]
@roshanr10 damn thanks... I was running 10 with nvm but I needed to uninstall the package and installed again, since previously it was installed with 12 and that affected it :)

**Comment by tristan1990** [comment]
I uninstalled node completely, then reinstalled 0.10.36, installed `brew install ffmpeg` and all is working great. Good work!
